index.bs

```diff
@@ -2289,10 +2297,33 @@ during registration.
     :   {{AuthenticatorAttestationResponse/getTransports()}}
     ::  This operation returns the value of {{AuthenticatorAttestationResponse/[[transports]]}}.
 
+    :   {{AuthenticatorAttestationResponse/getAuthenticatorData()}}
+    ::  This operation returns the [=authenticator data=] contained within {{AuthenticatorAttestationResponse/attestationObject}}. See [[#sctn-public-key-easy]].
+
+    :   {{AuthenticatorAttestationResponse/getPublicKey()}}
+    ::  This operation returns a string that contains the base64-encoded, DER [=SubjectPublicKeyInfo=] of the new credential, or an empty string if this is not available. See [[#sctn-public-key-easy]].
```

Comment:
Are we sure empty string is the best way to signal "not supported"? What about `null`, `undefined` or `false`? Changing the type is a bit ugly, but I think it's usually better than special sentinel values of the same type.

How about `getAuthenticatorData()` and `getPublicKey()` both return `ArrayBuffer` or both return base64(url?) `DOMString`? We use `ArrayBuffer` for all other byte arrays, right?

Reply:
I vote for ArrayBuffer, and make it nullable rather than checking len>0.

Reply:
Will make it nullable.